pom.xml

```diff
@@ -21,7 +21,7 @@
 
     <properties>
         <java.version>21</java.version>
-        <thymeleaf-layout-dialect.version>3.3.0</thymeleaf-layout-dialect.version>
+        <thymeleaf-layout-dialect.version>3.4.0</thymeleaf-layout-dialect.version>
```

Comment:
This version is fixing a bug with `th:with` which actually was malfunctioning, before this upgrade.
<img width="935" height="331" alt="Screenshot 2025-07-18 at 10 58 07" src="https://github.com/user-attachments/assets/93434b04-3ebe-4e94-8d53-ec1cd9c4614a" />